lib/vmdb/util.rb

```diff
@@ -3,18 +3,13 @@ module Util
     def self.http_proxy_uri(proxy_config = :default)
       # TODO: (julian) This looks really messy. Need to make it look nicer.
 
-      vmdb_proxy = VMDB::Config.new("vmdb").config[:http_proxy]
-
-      return nil unless vmdb_proxy
-
-      if vmdb_proxy[proxy_config].nil?
+      if ::Settings.http_proxy[proxy_config].nil?
         $log.warn("Could not find proxy setting for #{proxy_config}")
+        proxy = ::Settings.http_proxy.to_h
+      else
+        proxy = ::Settings.http_proxy[proxy_config].to_h
       end
```

Comment:
I was trying to keep ```warn``` message, but looks like there is no much sense in keeping that ```log.warn```. 
Will simplify...